app/models/pusher.rb

```diff
@@ -110,7 +110,7 @@ def update_remote_bundler_api(to = RestClient)
     }.to_json
 
     begin
-      timeout(5) do
+      ::Timeout.timeout(5) do
         to.post @bundler_api_url,
           json,
           :timeout        => 5,
```

Comment:
Right! so if we set the timeout on the client, there is no need to use an extra timeout here I think.
Not sure why it was it here in the first place?.?